raiden/transfer/mediated_transfer/mediation_fee.py

```diff
@@ -37,6 +38,109 @@ def __call__(self, x: float) -> float:
         i = bisect_right(self.x_list, x) - 1
         return self.y_list[i] + self.slopes[i] * (x - self.x_list[i])
 
+    def __repr__(self) -> str:
```

Comment:
Interpolate's constructor is missing a return type `-> None:`

Reply:
Mypy assumes "-> None" for `__init__`. It's only required when no arguments are taken, to distinguish the case of typed vs untyped constructor. I can certainly add one for consistency if you prefer, but I personally wouldn't.

Reply:
Please just add it for consistency. We also got it in the contributing guide.

Reply:
Sure, I didn't notice it in the guide (or I forgot.)